templates/partials/committee/receipts-tab.html

```diff
@@ -6,7 +6,7 @@
   <div class="container">
     <div class="section__heading">
       <h2 class="heading__title" id="section-2-heading">
-        Money received by the <span class="term" data-term="Committee">committee</span>
+        Money received by the committee - {{ cycle|fmt_year_range }}
```

Comment:
Good question. I think it makes sense to not make words in headings glossary links. @emileighoutlaw ?